FESS-GUI/setpassworddialog.cpp

```diff
@@ -14,6 +15,7 @@ SetPasswordDialog::SetPasswordDialog(QWidget *parent) :
     ui(new Ui::SetPasswordDialog)
 {
     ui->setupUi(this);
+    this->setWindowTitle(PASSWORD_RESET_DIALOG);
```

Comment:
But it's already set in the .ui file. I just don't see why we should define all names in one file if each one is only used in one place. We aren't planning to change them en masse, are we? For instance, to translate it all to a different language. That would be the only reason I see to put them all in one file.